### PR TITLE
Solve issue #4206: The contrast of font color of code comment in code cell is lower than 4.5:1

### PR DESCRIPTION
**Pull request by taohan16**
Overwrite the highlight-comment style class with color #007979

### CODE REVIEW

**Comment by taohan16** [comment]
Before
color contrast: 4.2:1
After
color contrast: 4.8:1

**Comment by takluyver** [comment]
Is it possible you've used the wrong 'after' screenshot?

File: Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
```
Comment (takluyver) [inline Untitled.ipynb]: I guess this was committed accidentally. :-)

File: notebook/static/notebook/less/highlight.less [hunk 2]
```diff
@@ -64,6 +64,12 @@ Adapted from GitHub theme
   color: #AA22FF;
 }
 
+//fix issue #4206, overwrite .highlight-comment class
+.highlight-comment{
+  color:#007979;
```
Comment (takluyver) [inline notebook/static/notebook/less/highlight.less]: Do we need to add a new rule? Can we not modify the existing one at line 32?

Comment (taohan16) [reply notebook/static/notebook/less/highlight.less]: Sorry for the mistakes made, I had updated the screenshots and code now.

Comment (taohan16) [reply notebook/static/notebook/less/highlight.less]: > Do we need to add a new rule? Can we not modify the existing one at line 32?

Sorry for the mistakes made, I had updated the screenshots and code now.

**Pull request merged**